### PR TITLE
misc: update easter egg emojis

### PR DESCRIPTION
**Pull request by canyie**
# Update easter egg emojis

(Not tested, CI can't run for my fork)

## Description

- The transgender emoji is also common and the transgender flag emoji includes it
- The "fish cake with swirl" emoji is also widely used among Chinese MtFs/MtXs

## Issues Fixed or Closed by This PR
N/A

## Check List

- [ ] I have tested the changes and verified that they work and don't break anything(as well as I can manage) or drop the support for previous versions.
- [x] I have merged commits that are meaningless for follow-up work and confirmed that they will not cause damage to follow-up maintenance

### CODE REVIEW

**Comment by NextAlone** [comment]
**喵**

File: app/src/main/java/io/github/qauxv/fragment/SettingsMainFragment.kt [hunk 1]
```diff
@@ -190,8 +190,8 @@ class SettingsMainFragment : BaseRootLayoutFragment() {
     override fun onResume() {
         super.onResume()
         try {
-            if (ContactUtils.getBuddyName(AppRuntimeHelper.getAppRuntime()!!, AppRuntimeHelper.getAccount())
-                    ?.contains("\uD83C\uDFF3\uFE0F\u200D\u26A7\uFE0F") == true &&
+            val buddyName = ContactUtils.getBuddyName(AppRuntimeHelper.getAppRuntime()!!, AppRuntimeHelper.getAccount())
+            if ((buddyName?.contains("\u26A7\uFE0F") == true || buddyName?.contains("\uD83C\uDF65") == true) &&
```
Comment (Cryolitia) [inline app/src/main/java/io/github/qauxv/fragment/SettingsMainFragment.kt:194]:
```suggestion
            if ((buddyName?.contains("\uD83C\uDFF3\uFE0F\u200D\u26A7\uFE0F") == true || buddyName?.contains("\u26A7\uFE0F") == true || buddyName?.contains("\uD83C\uDF65") == true) &&
```

File: app/src/main/java/me/singleneuron/util/EasterEggsForTheNonexistent.kt [hunk 2]
```diff
@@ -44,7 +44,7 @@ fun processSearchEasterEgg(text: String, context: Context) {
 
 private val easterEggsMap by lazy {
     hashMapOf(
-        arrayOf("\uD83C\uDFF3\uFE0F\u200D\u26A7\uFE0F", "\uD38C\uDF65", "mtf", "mtx", "ftm", "ftx", "transgender") to forSuBanXia,
+        arrayOf("\u26A7\uFE0F", "\uD38C\uDF65", "mtf", "mtx", "ftm", "ftx", "transgender") to forSuBanXia,
```
Comment (Cryolitia) [inline app/src/main/java/me/singleneuron/util/EasterEggsForTheNonexistent.kt:47]:
```suggestion
        arrayOf("\uD83C\uDFF3\uFE0F\u200D\u26A7\uFE0F", "\u26A7\uFE0F", "\uD38C\uDF65", "mtf", "mtx", "ftm", "ftx", "transgender") to forSuBanXia,
```

**Review by Cryolitia — CHANGES_REQUESTED** [review]
🏳‍⚧作为一个有独立Unicode编码的符号是不需要将其编码拆分成两部分再分别判断的，是你的设备无法打出跨性别旗吗？

File: app/src/main/java/io/github/qauxv/fragment/SettingsMainFragment.kt (hunk 1 above)
Comment (canyie) [reply app/src/main/java/io/github/qauxv/fragment/SettingsMainFragment.kt:194]: 跨性别旗🏳‍⚧实际上是由一个白旗🏳️+一个[零宽连字符](https://emojipedia.org/zero-width-joiner/)+一个跨性别符号⚧️组成的，只判断跨性别符号理论上也可以识别到跨性别旗

**Review by Cryolitia — APPROVED** [review]
image
这个地方没问题了，可以准备合并。
为什么设置为draft？

**Pull request merged**